docs/Single-server-VictoriaMetrics.md

```diff
@@ -1,13 +1,9 @@
 ---
 sort: 1
 weight: 1
-menu:
```

Comment:
this and following changes doesn't look relevant

Reply:
that's why I mentioned @tenmozes